client/src/components/RelatedItemsAndComparisons/styles.css

```diff
@@ -9,18 +9,58 @@
   margin: auto;
```

Comment:
Just something to discuss, we should probably decide as a group on css naming conventions. Melissa used BEM and half of mine is in BEM other half is CamelCase lol.